bleachbit/Windows.py

```diff
@@ -604,40 +605,105 @@ def symlink_or_copy(src, dst):
+        wndClass.hInstance      = hInstance
+        wndClass.hIcon          = win32gui.LoadIcon(0, win32con.IDI_APPLICATION)
+        wndClass.hCursor        = win32gui.LoadCursor(0, win32con.IDC_ARROW)
+        wndClass.hbrBackground  = win32gui.GetStockObject(win32con.WHITE_BRUSH)
+        wndClass.lpszClassName  = className
+
+        # register window class
+        wndClassAtom = None
         try:
-            import locale
-            lang_id = locale.getdefaultlocale()[0].split('_')[0]
-        except Exeption as e:
-            logger.exception('cannot find lang_id')
-            lang_id = '??'
-        logger.debug('detected lang_id=%s', lang_id)
-        extra_fonts = {}
-        extra_fonts['ko'] = ['malgun.ttf', 'malgunbd.ttf', 'malgunsl.ttf']
-        extra_fonts['hi'] = ['mangal.ttf', 'mangalb.ttf']
-        extra_fonts['ja'] = ['meiryo.ttc', 'meiryob.ttc', 'msgothic.ttc']
-        extra_fonts['zh'] = ['msyh.ttc', 'msyhbd.ttc']
-        if lang_id in extra_fonts:
-            fonts_needed.extend(extra_fonts[lang_id])
-        for font in fonts_needed:
-            dst_font_fn = os.path.join(_GTK_FONTS_FOLDER, font)
-            src_font_fn = os.path.join(windows_fonts_folder, font)
-            if not os.path.exists(src_font_fn):
-                logger.error('the font file does not exist: %s', src_font_fn)
-            elif not os.path.exists(dst_font_fn):
-                symlink_or_copy(src_font_fn, dst_font_fn)
+            wndClassAtom = win32gui.RegisterClass(wndClass)
+        except Exception as e:
+            print(e)
+            raise e
+
+        displayWidth = win32api.GetSystemMetrics(0)
+        displayHeigh = win32api.GetSystemMetrics(1)
+        windowWidth = displayWidth // 4
+        windowHeight = displayHeigh // 4
+        windowPosX = (displayWidth - windowWidth) // 2
+        windowPosY = (displayHeigh - windowHeight) // 2
+
+        hWindow = win32gui.CreateWindow(
+            wndClassAtom,                   #it seems message dispatching only works with the atom, not the class name
+            'Bleachbit splash screen',
+            win32con.WS_POPUPWINDOW | win32con.WS_VISIBLE,
+            windowPosX,
+            windowPosY,
+            windowWidth,
+            windowHeight,
+            0,
+            0,
+            hInstance,
+            None)
+
+        # Show & update the window
+        win32gui.ShowWindow(hWindow, win32con.SW_SHOW)
+        # win32gui.UpdateWindow(hWindow)
+        win32gui.SetActiveWindow(hWindow)
+        win32gui.BringWindowToTop(hWindow)
+
+        return hWindow
+
+    @staticmethod
+    def wndProc(hWnd, message, wParam, lParam):
+
+        if message == win32con.WM_PAINT:
+            hDC, paintStruct = win32gui.BeginPaint(hWnd)
+
+            rect = win32gui.GetClientRect(hWnd)
+            vcenter_rect = (0, rect[3] // 2 - 50, rect[2], rect[3])
+            win32gui.DrawText(
+                hDC,
+                ("Welcome to Bleachbit, it is loading fonts.\n"
```

Comment:
@rados 
This is informative text, but for now please let's go with simple instead like "BleachBit is starting".  

If there could be a progress bar or spinner, I would make the text simpler, and many Android apps I tested use just a logo. Let's not worry about these for now, though.